releasenotes/notes/properties-last-update-date-3d09884ff8634a2f.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    The `last_update_date` parameter for
+    :class:`~qiskit.providers.models.BackendProperties` can now be either a
+    ``datetime`` object or a datetime string (in ISO format).
```

Comment:
Actually I don't think we need this feature release note since this is for compatibility with the previous marshmallow implementation. But it's not worth blocking over, I'll catch this in the pre-release release note round up

Reply:
I don't think marshmallow accepted a `datetime` object.